app/enterprise/2.2.x/developer-portal/administration/application-registration/auth-provider-strategy.md

```diff
@@ -0,0 +1,123 @@
+    and client ID credential, hybrid mode is also not supported for the Key
+    Authentication plugin. If your organization uses hybrid mode, you must use an external identity
+    provider and configure the Kong OIDC plugin.
+  </div>
+
+- When an external OAuth2 is the system of record, the Application Registration
+  plugin works in conjunction with the Kong OIDC plugin.
+
+The third-party authorization strategy (`external-oauth2`) applies to all
+applications across all Workspaces (Dev Portals) in a Kong cluster.
+
+### Authorization provider strategy configuration for Application Registration {#portal-app-auth}
+
+The `portal_app_auth` configuration option must be set in `kong.conf` to enable
+the Developer Portal Application Registration plugin with your chosen
+authorization strategy. The default setting (`kong-oauth2`) accommodates the Kong
+OAuth2 or Kong Key Authentication plugins.
+
+Available options:
+
+* `kong-oauth2`: Default. Kong is the system of record. The Application
+  Registration plugin is used in conjunction with the Kong OAuth2 or
+  Kong Key Authentication plugin. The `kong-oauth2` option can only be used with
+  classic (traditional) deployments. Because the OAuth2 plugin requires a database
+  for every Kong instance, the `kong-oauth2` option cannot be used with hybrid mode
+  deployments.
+* `external-oauth2`: An external IdP is the system of record. The
+  Portal Application Registration plugin is used in conjunction with the Kong
+  OIDC plugin. The `external-oauth2` option can be used with any deployment type.
+  The `external-oauth2` option must be used with
+  [hybrid mode](/enterprise/{{page.kong_version}}/deployment/hybrid-mode/)
+  deployments because hybrid mode does not support `kong-oauth2`.
+
+#### Set external portal authentication {#set-external-oauth2}
+
+If you are using an external IdP, follow these steps.
+
+1. Open `kong.conf.default` and set the `portal_app_auth` option to your chosen
+   strategy. The example configuration below switches from the default
+   (`kong-oauth2`) to an external IdP (`external-oauth2`).
+
+   ```
+   portal_app_auth = external-oauth2
+            # Developer Portal application registration
+            # auth provider and strategy. Must be set to configure
+            # authentication in conjunction with the application_registration plugin.
+            # Currently accepts kong-oauth2 or external-oauth2.
+   ```
+
+2. [Restart](https://docs.konghq.com/2.1.x/cli/#kong-restart) your Kong Enterprise
+   instance.
+
+### Next Steps
+
+1. Enable the [Application Registration plugin](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/enable-application-registration) on a Service.
+
+2. Enable a supported authentication plugin on the same Service as the Application Registration plugin,
+   as appropriate for your authorization strategy.
+
+    Kong OAuth2 strategy:
+
+    * If using the Kong-managed authorization strategy
```

Comment:
Kong-managed implies that Kong the company manages the strategy. What you want to say is that Kong Gateway manages it, correct?

Reply:
yeah will rework